tools/updtset.cmd

```diff
@@ -0,0 +1,235 @@
+@echo off
+setlocal EnableDelayedExpansion
+setlocal EnableExtensions
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::ver 1.13 / 2022-06-29::::
+:: this script does following:
+:: - compose tileset from the source
+:: - put composed tileset into some folder for future use
+:: - copy said composed tileset from the folder into the game so you can try in on the fly
+:: - copy layering json file from tileset source into the game
+::
+:: PLEASE setup 5 variables below, put correct text inside a quotes
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: 1. Set correct folder same as in github desktop repository local path
+
+set tileset_fork="T:\cdda-tilesets"
+
+::
+:: 2. Set correct tileset folder name (e.g. UltimateCataclysm)
+
+set def_tileset="UltimateCataclysm"
+
+::
+:: 3. Set path to compose.py file 
+::    if it is in cdda source local repository type full path to it
+::    or if it's in your workshop type this path
```

Comment:
```suggestion
```
This part is confusing, no matter where compose.py is you set the path to it, no need to describe special cases